design-library/src/components/BccProgress/BccProgress.css

```diff
@@ -14,17 +14,21 @@
 
     /* Size */
     .bcc-progress-container .bcc-form-label-lg .bcc-form-label-optional {
-        @apply text-label;
+        @apply text-label-base;
```

Comment:
This class no longer exists. The typography styles were updated, which is the main reason we'll be tagging a 2.0 release soon. See this page in the docs for the new classes: https://design-library-dev.developer.bcc.no/?path=/docs/tokens-typography--docs

You need to rebase your branch on the latest `main`. It might be easier to start a new branch because otherwise it will also try to rebase all your older commits that were already merged into main. Or maybe drop all the old commits when rebasing (let me know if you don't know how to proceed here, happy to help as rebasing in Git can get fairly complicated).